src/Controller/MyGengoController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\tmgmt_mygengo\Controller\MyGengoController.
+ */
+
+namespace Drupal\tmgmt_mygengo\Controller;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\tmgmt\Entity\Job;
+use Drupal\tmgmt\Entity\JobItem;
+use Symfony\Component\HttpFoundation\Response;
+use Drupal\tmgmt\Entity\RemoteMapping;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Route controller class for the tmgmt translator entity.
+ */
+class MyGengoController extends ControllerBase {
+
+  /**
+   * Process response from mygengo.
+   */
+  function callback() {
+    // First, check if this is a comment notification.
+    if (isset($_POST['comment'])) {
```

Comment:
Pass Request $request to the method and then load data from there: $request->request->get('comment') and so on.